src/main/java/edu/umich/tl/DeleteUtils.java

```diff
@@ -0,0 +1,69 @@
+package edu.umich.tl;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.util.Properties;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+
+
+
+public class DeleteUtils {
```

Comment:
Poorly named class - this class has nothing to do with deleting and it's so short, I'd just add the methods as private methods to the CourseDelete class.